functions/src/events/scrapeEvents.ts

```diff
@@ -136,6 +138,45 @@ class SessionScraper extends EventScraper<SessionContent, Session> {
   }
 }
 
+const extractAudioFromVideo = async (
+  EventId: number,
+  videoUrl: string
+): Promise<string> => {
+  const audioBuffer = await new Promise<Buffer>((resolve, reject) => {
+    ffmpeg()
+      .input(videoUrl)
+      .toFormat("mp3")
+      .on("end", () => resolve(audioBuffer))
+      .on("error", reject)
+      .pipe()
+  })
+
+  const bucket = admin.storage().bucket()
```

Comment:
nit: We also export `storage` from our firebase.ts file, so we can use that here.